vendor.conf

```diff
@@ -42,3 +42,4 @@ golang.org/x/text 19e51611da83d6be54ddafce4a4af510cb3e9ea4
 github.com/dmcgowan/go-tar go1.10
 github.com/stevvooe/ttrpc d2710463e497617f16f26d1e715a3308609e7982
 github.com/syndtr/gocapability db04d3cc01c8b54962a58ec7e491717d06cfcc16
+github.com/containerd/zfs 2e6f60521b5690bf2f265c416a42b251c2a3ec8e
```

Comment:
Please vendor github.com/mistifyio/go-zfs as well

Reply:
How is CI passing without this dependency being included?

Reply:
@dnephin failing

Reply:
Why do we need to two zfs packages?